src/index.ts

```diff
@@ -131,6 +130,14 @@ export class AssembleSubmitButtonExtension
           void request<void>('auth/token', {
             method: 'POST'
           });
+
+          let authIntervalId;
+          authIntervalId = setInterval(
+            authCallback,
+            500,
```

Comment:
magic number: if this is one half second, you can express it as `0.5 * ONE_SECOND` using the module-level constant